src/anyp/TrafficMode.h

```diff
@@ -46,7 +59,7 @@ class TrafficMode
      *  - URL translation from relative to absolute form
      *  - Same-Origin verification is mandatory
      *  - destination pinning is recommended
-     *  - authentication prohibited
+     *  - proxy authentication prohibited
```

Comment:
I decided to adjust these lines to be consistent with cf.data.pre documentation for 'proxy_auth':
```
NOTE: proxy_auth can't be used in a transparent/intercepting proxy
```

Reply:
I fixed 'proxy auth' with a better 'Squid auth' at c0992c1.